integration/utk_test.go

```diff
@@ -101,3 +105,53 @@ func TestExtractAssembleExtract(t *testing.T) {
 		})
 	}
 }
+
+// TestRegressionJson tests for regression in the JSON. After making a change
+// which affects the tree, you must commit changes to the golden JSON files
+// with:
+//
+//     utk integration/roms/OVMF.rom json > integration/roms/OVMF.json
+//
+// Otherwise, this test will fail. This gives you a chance to review how your
+// code affects the tree and identify any mistakes.
+func TestRegressionJson(t *testing.T) {
+	// Create a temporary directory.
+	tmpDir := createTempDir(t)
+	defer os.RemoveAll(tmpDir)
+
+	for _, tt := range romList(t) {
```

Comment:
how about you iterate over the jsonlist for this test? so we can have roms without the golden json, and this test only runs if you have those

Reply:
Ok, sounds good!